src/components/RentalCard/RentalCard.css

```diff
@@ -1,55 +1,61 @@
 .rentalCardContainer {
-    background-color: white;
-    box-shadow: 0px 3px 5px rgba(0, 0, 0, 0.05);
-
-
-    height: 278px;
-    width: 13.625em;
-    
-    border-radius: 5px;
-    margin: 40px;
-    box-sizing: border-box;
-    /* border: 0px solid #333333; */
-    transition: box-shadow 0.6s;
+  background-color: white;
+  box-shadow: 0px 3px 5px rgba(0, 0, 0, 0.05);
+
```

Comment:
I'll answer here regarding all CSS as a whole: 

Thank you for all the inputs you gave on the styles. Being honest, the CSS part is the part I am less "confident" about. Meaning: How CSS really works and how specific CSS styles work.